vsts/CredScanSuppressions.json

```diff
@@ -0,0 +1,37 @@
+{
+    "tool": "Credential Scanner",
+    "suppressions": [
+        {
+            "placeholder": "0000000000000000000000000000000000000000000=",
+            "_justification": "Dummy secret used in unit tests, it is fake"
+        },
+        {
+            "placeholder": "1111111111111111111111111111111111111111111=",
+            "_justification": "Dummy secret used in unit tests, it is fake"
+        },
+        {
+            "placeholder": "2222222222222222222222222222222222222222222=",
+            "_justification": "Dummy secret used in unit tests, it is fake"
+        },
+        {
+            "placeholder": "3333333333333333333333333333333333333333333=",
+            "_justification": "Dummy secret used in unit tests, it is fake"
+        },
+        {
+            "placeholder": "4444444444444444444444444444444444444444444=",
+            "_justification": "Dummy secret used in unit tests, it is fake"
+        },
+        {
+            "placeholder": "5555555555555555555555555555555555555555555=",
+            "_justification": "Dummy secret used in unit tests, it is fake"
+        },
+        {
+            "placeholder": "notsafe",
+            "_justification": "Dummy secret used in unit tests, it is fake"
+        },
+        {
+            "file": ["\\device\\external_modules\\dotnetty\\shared\\dotnetty.com.pfx", "\\device\\external_modules\\dotnetty\\shared\\contoso.com.pfx "],
+            "_justification": "The PFX certificate is required for building the UWP Sample"
+        }
```

Comment:
Is this needed during runtime? Why can't you leave it at "Your PFX cert here" as a string placeholder?

Reply:
These are required by dotnetty tests and examples, which are included in our app as an external module. Since the certs and their references are both a part of the dotnetty repo, we have explicitly suppress those occurrences.